package.json

```
@@ -13,8 +13,18 @@
   },
   "license": "MIT",
   "contributors": [
-    "Marine Dunstetter",
-    "Guillaume Gérard"
+    {
+      "name": "Marine Dunstetter",
+      "email": "marine.dunstetter@simplabs.com"
```

> I have used what is publicly available

Then I would say "good catch" 😅 

```suggestion
      "email": "marine.dunstetter@mainmatter.com"
```